app/policies/benchmark_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Policies for accessing Benchmarks
+class BenchmarkPolicy < ApplicationPolicy
+  def index?
+    true
```

Comment:
I raised this on the wizard frontend PR, but today we allow users to upload any random SSG version and benchmark. This would allow users from different accounts to see and use those benchmarks, which I don't think is what we want. It seems we may need to associate some benchmarks with accounts in the future (we don't need to fix this in this PR necessarily).

Reply:
That's a good point, we should think about this though, because SSG should be visible to everyone but custom benchmarks should not...